python/module.hpp

```diff
@@ -378,6 +395,7 @@ py::class_<T> eos_class(py::module_ & m, std::string name) {
     .def("BulkModulusFromDensityInternalEnergy", &two_params<T, &T::BulkModulusFromDensityInternalEnergy>, py::arg("rho"), py::arg("sie"), py::arg("lmbda"))
     .def("GruneisenParamFromDensityTemperature", &two_params<T, &T::GruneisenParamFromDensityTemperature>, py::arg("rho"), py::arg("temperature"), py::arg("lmbda"))
     .def("GruneisenParamFromDensityInternalEnergy", &two_params<T, &T::GruneisenParamFromDensityInternalEnergy>, py::arg("rho"), py::arg("sie"), py::arg("lmbda"))
+    //.def("InternalEnergyFromDensityPressure", &three_params<T, &T::InternalEnergyFromDensityPressure>, py::arg("rho"), py::arg("P"), py::arg("sie"), py::arg("lmbda"))
```

Comment:
This is the line (along with the one below) that fails

Reply:
If you have a function that corresponds to the two_params function pointer format, that should work.